bin/templates/scripts/cordova/lib/Podfile.js

```diff
@@ -98,8 +98,8 @@ Podfile.prototype.getTemplate = function () {
     var projectName = this.escapeSingleQuotes(this.projectName);
     return util.format(
         '# DO NOT MODIFY -- auto-generated by Apache Cordova\n' +
-        'platform :ios, \'8.0\'\n' +
-        'target \'%s\' do\n' +
+        'platform :ios, \'%s\'\n' +
+        'use_frameworks!\n' +
```

Comment:
we can't use `use_frameworks!`. See: https://issues.apache.org/jira/browse/CB-11893